teacher_class.rb

```diff
@@ -0,0 +1,14 @@
+require_relative 'person_class'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(id, age, specialization, name = 'UNKNOWN')
+    super(id, name, age)
```

Comment:
- [ ] Kindly use the correct arrangement of parameters in the `super` method. `age` should come before `name`.

  ```
  super(id, age, name)
  ```